lib/smart-survey-api.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const axios = require('axios');
+const querystring = require('querystring');
+const SmartSurveyClient = require('smartsurvey-client');
+
+module.exports = {
+  getResponses: (baseUrl, surveyID, endPoint, token, tokenSecret) => {
+    querystring.stringify({ foo: 'bar', baz: ['qux', 'quux'], corge: '' });
+    const apiUrlPart = querystring.stringify({'api_token': token, 'api_token_secret': tokenSecret});
+    axios.get(`${baseUrl}/${surveyID}/${endPoint}?${apiUrlPart}`)
```

Comment:
Is there any harm keeping both? There's an api call that isn't covered for SmartSurveyClient where I need to use axios.  

I know you suggested putting in a PR for SmartSurveyClient.  That I'll have to put in as a later stage as I consider this PR out of scope for that.